typhos/display.py

```diff
@@ -879,6 +883,9 @@ def add_device(self, device):
         if not self.label.text():
             self.label.setText(device.name)
 
+        if not self.notes_edit.text():
+            self.notes_edit.setup_data(device.name)
```

Comment:
Any thoughts on running this more often/periodically/when the file updates? I think this falls out of sync fairly quickly.

Reply:
You're thinking for long-running screens?  Or if people edit the file manually and want the screen to update?  I'm not too big a fan of watching files, but I suppose we could run it periodically.  

I imagined a world where people only edit this via GUI apps, but I guess I can't assume that's reality

Reply:
Imagine I open the screen on tmo-control, then you open it on tmo-daq, then James opens it on tmo-console. The changes can't propagate between these copies of the screen without a restart.